.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: python
 
+services:
+  - docker
+
 python:
   - "3.6"
 
 script:
   - python setup.py flake8
+  - cp docker/.env.example docker/.env
+  - docker build . -t pyslackers/sirbot.pythondev -f docker/Dockerfile
+  - docker-compose -f docker/docker-compose.ngrok.yml build
+  - docker-compose -f docker/docker-compose.yml build
```

Comment:
Since the `image` and `build` are defined in your `docker-compose.yml` you can remove the previous two docker commands, compose will build and tag them